test/e2e/storage/vsphere/vsphere_volume_vsan_policy.go

```diff
@@ -341,3 +339,22 @@ func invokeStaleDummyVMTestWithStoragePolicy(client clientset.Interface, masterN
 	framework.ExpectNoError(err)
 	framework.ExpectEqual(isVMPresentFlag, false, errorMsg)
 }
+
+func getControlPlaneNode(client clientset.Interface) (string, error) {
+	regKubeScheduler := regexp.MustCompile("kube-scheduler-.*")
+	regKubeControllerManager := regexp.MustCompile("kube-controller-manager-.*")
```

Comment:
as mentioned above, i don't understand why this test needs a control-plane node (so maybe we have to clear this topic first).

but a meta-question here is - if a node in a certain setup is running the scheduler and the KCM, but not the kube-apiserver (GCP used to do that) can we still assume that this is a "control-plane" node?